lib/marked.js

```diff
@@ -926,6 +926,7 @@
       }
 
       var match = this.rules.inline.strong.start.exec(src);
+      console.log(match);
```

Comment:
Remove console.log()

Reply:
lib will get regenerated once it is merged but /lib and min scripts should probably be removed from the PR

Reply:
I forgot to exclude the libs. I will do that.

Reply:
I undid the changes on the lib files so they aren't part of the PR.